### PR TITLE
Add missing derivatives.yaml input

### PR DESCRIPTION
**Pull request by zdevito**

### CODE REVIEW

**Comment by zdevito** [comment]
#4079

File: tools/setup_helpers/generate_code.py [hunk 1]
```diff
@@ -17,7 +17,8 @@ def all_generator_source():
 inputs = [
     'torch/csrc/generic/TensorMethods.cwrap',
     'torch/lib/tmp_install/share/ATen/Declarations.yaml',
-    'torch/lib/tmp_install/share/ATen/Declarations.yaml'
+    'torch/lib/tmp_install/share/ATen/Declarations.yaml',
```
Comment (colesbury) [inline tools/setup_helpers/generate_code.py]: Why is this listed twice?

**Review by colesbury — COMMENTED** [review]
Should this include all the files in the directory torch/csrc/generic/methods?

**Comment by zdevito** [comment]
I added them. Hopefully that is all the deps. I added some logic to the aten build last week to _force_ new dependencies to be explicitly reflected in the build, but I am not sure it is worth doing for all of the generate code in pytorch.

**Review by colesbury — APPROVED** [review]
lgtm

**Comment by ezyang** [comment]
Thanks!

**Pull request merged**